src/components/articles/IconLink.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+function IconLink(props) {
+   const {link, title, icon} = props;
+   return (
+      <a href={link} target={"_blank"} rel="noopener noreferrer">
+         <i className={icon}/> {title}
+      </a>
```

Comment:
**Suggestion:** Consider adding a 'title' attribute to the anchor tag. This will provide additional information about the link and improve accessibility.
```suggestion
      <a href={link} target={"_blank"} rel="noopener noreferrer" title={title}>
        <i className={icon}/> {title}
      </a>
```